src/AppActions.js

```diff
@@ -14,6 +14,12 @@ const setData = async (url, headers, options) => {
     return response.data;
 };
 
+const deleteData = async (url, headers, options) => {
```

Comment:
ok, new plan, lets not add any actions for this one off table, they won't be reused anywhere else, instead take a look at the ViewHostAcks.js line 30, do something like that in the AddEditTopic modal (the `await API.delete(`${BASE_URL}/hostack/${host.id}/`);` thing)

ALSO, talked with @dkuc he reports two thingsssss
* updating a topic has to be done over PUT not post (and will include the slug in the url)
* what the mocks call `label` needs to be `tag`  (the one that doesn't autofill) and its gotta be passed when creating/updating as well

Reply:
Should the API calls be nested in a try-catch like they are in the ViewHostAcks modal?